DEHPMatlab/Services/MatlabConnector/MatlabConnector.cs

```diff
@@ -91,65 +92,67 @@ public MatlabConnector(IStatusBarControlViewModel statusBarControl)
         /// Launch a Matlab Instance and connects to it
         /// </summary>
         /// <param name="comInteropName">The name of the COM Interop</param>
-        public void Connect(string comInteropName)
+        /// <returns>The <see cref="Task"/></returns>
+        public async Task Connect(string comInteropName)
```

Comment:
For all the mothods in this class I am not certain why do you want to do internal `Task.Run()` if they are already awaited upstream? Dont you do then double `await`? As in somewhere you already call `await MatlabConnector.Connect(string)` no?